experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -336,6 +344,13 @@ function Interfacer.update_field!(sim::ClimaAtmosSimulation, ::Val{:turbulent_fl
     parent(sim.integrator.p.precomputed.sfc_conditions.ρ_flux_q_tot) .=
         parent(F_turb_moisture) .* parent(surface_normal) # (evap)
 
+    # If available, also Monin-Obukhov quantities
+    if :L_MO in propertynames(fields)
```

Comment:
I thought Monin Obukhov quantities will always be available when running with PartitionedStateFluxes. Is this to support CombinedStateFluxes, or does PartitionedFluxes also work with `BulkScheme` now?

Reply:
I guess that's true, I can just assume they are there.

I was probably tried to get more compatibility with other schemes